repos.d/apkfission.yaml

```diff
@@ -0,0 +1,40 @@
+###########################################################################
+# APK Fission packages
+#
+# https://apkfission.net/install.html
+###########################################################################
+
+{% macro apkfission(version, subrepos=['main','nonfree'], archs=['aarch64','armv7','pmmx','ppc','ppc64','x86_64']) %}
```

Comment:
Webapp does not currently handle a lot of similar packages, e.g. a package of the same thing for a lot of arches. That's why we only stich to ppc64 for adelie currently. I'd prefer the same here for now.

Reply:
Unfortunately, APK Fission's packaging has numerous differences between archs (firmware specific to aarch64, microcode specific to pmmx/x86_64, etc.), so I do need to have arch-related subrepos. I could pin to x86_64 for now since that's necessary, but I'd really like to not have to.